frontend/src/styles/main.scss

```diff
@@ -6,8 +6,10 @@ $link-color: #0d6efd;
 
 @import "./bootstrap";
 @import "flatpickr/dist/flatpickr.css";
-@import "./effects";
 
+#root {
+    overflow: hidden;
+}
```

Comment:
Hm, this is a pretty big rule to bust out here.  I'd prefer to figure out what's overflowing and fix that rather than hiding it.

I'm assuming this is for the prevent side-scroll card?  I'd played with it on iOS but couldn't see any overflow,  I do have an older android phone for testing though and was going to attempt that.  I can do so and let you know what I find out